time-based-packet-transmission-offload.md

```diff
@@ -95,16 +98,21 @@ if(setsockopt(sock, SOL_SOCKET, SO_TXTIME_SCHED, (char*)&optVal, optLen) == SOCK
 
 ## Sending Packets
 The app calls `WSASendMsg`.   
-The app passes ancillary data in the form of `TPTO_ANCILLARY_DATA`
+The app passes ancillary data in the form of `TPTO_ANCILLARY_DATA`.
 ```C
 typedef struct _TPTO_ANCILLARY_DATA {
     uint64_t TxTime;
-    uint8_t  DropIfRate
+    uint64_t TimeDelta;
 } TPTO_ANCILLARY_DATA;
 ```
 
 - TxTime to indicate when the associated packet is sent
-- DropIfRate to indicate whether to drop packet if it cannot be transmitted by the given deadline
+- TimeDelta is to spread batching packets
```

Comment:
"TimeDelta is to spread batching packets"- what does this mean?

Reply:
This is for LSO/USO to transmit segmented packet with interval of TimeDelta